src/Language/DifferentialDatalog/OVSDB/Compile.hs

```diff
@@ -106,7 +109,7 @@ mkSubtable :: (MonadError String me) => Bool -> String -> String -> ComplexType
 mkSubtable isinput tname cname t@ComplexType{..} = do
     keytype <- mkBaseType keyComplexType
     valtype <- maybe (return []) (\v -> (\vt -> ["value:" <+> vt]) <$> mkBaseType v) valueComplexType
-    let cols = [ mkColName tname <> ": uuid"
+    let cols = [ "_uuid: uuid"
```

Comment:
do you want to assert that this column does not already exist? You should give an error message as soon as possible if it does.

Reply:
I do